src/views/Alteration.vue

```diff
@@ -62,17 +62,19 @@
         <court-order-poa
           id="court-order"
           :validate="getAppValidate"
+          :draftCourtOrderNumber="getCourtOrderNum"
+          :hasDraftPlanOfArrangement="getPlanOfArrangement"
```

Comment:
I recommend that the value be named "has...", otherwise it looks like it returns an object or something.

Reply:
ah right! Will fix and update on this end as well